css/shared/spacing.css

```diff
@@ -1766,3 +1766,55 @@
 .w-fit {
   width: fit-content;
 }
+
+/* spacers */
+
+.spacer-1rem {
+  width: 100%;
+  height: 1rem;
+}
+
+.spacer-2rem {
+  width: 100%;
+  height: 2rem;
+}
+
+.spacer-3rem {
+  width: 100%;
+  height: 3rem;
+}
+
+.spacer-4rem {
+  width: 100%;
+  height: 4rem;
+}
+
+.spacer-5rem {
+  width: 100%;
+  height: 5rem;
+}
+
+.spacer-6rem {
+  width: 100%;
+  height: 6rem;
+}
+
+.spacer-7rem {
+  width: 100%;
+  height: 7rem;
+}
+
+.spacer-8rem {
+  width: 100%;
+  height: 8rem;
+}
+
+.spacer-9rem {
+  width: 100%;
+  height: 9rem;
+}
+
+.spacer-10rem {
+  width: 100%;
+  height: 10rem;
+}
```

Comment:
Suggestion: We are using width in every space class here. It would be better to create a common utility class for this.

Reply:
Okay i will make the changes .